### PR TITLE
feat(ui): Improve monitor edit breadcrumbs

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Looks like this now

### CODE REVIEW

File: static/app/views/monitors/edit.tsx [hunk 1]
```diff
@@ -69,7 +72,21 @@ export default function EditMonitor() {
...
+                      <IdBadge
+                        project={monitor.project}
+                        avatarSize={16}
+                        hideName
+                        avatarProps={{hasTooltip: true, tooltip: monitor.project.slug}}
+                      />
+                      {monitor.name}
+                    </MonitorBreadcrumb>
+                  ),
+                  to: `/organizations/${organization.slug}/crons/${monitor.slug}/`,
```
Comment (davidenwang) [inline static/app/views/monitors/edit.tsx:86]: probably wont matter once we have the env selector in, but right now does it have the correct environment in the url?

Comment (evanpurkhiser) [reply static/app/views/monitors/edit.tsx:86]: oh does it persist that

Comment (evanpurkhiser) [reply static/app/views/monitors/edit.tsx:86]: yeah I see it 👍

Comment (davidenwang) [reply static/app/views/monitors/edit.tsx:86]: its okay tho lemme merge in my env selector and this should just work fine

Comment (evanpurkhiser) [reply static/app/views/monitors/edit.tsx:86]: 👍 cool

**Review by davidenwang — APPROVED** [review]

**Pull request merged**